kqueen/config/demo.py

```diff
@@ -3,7 +3,6 @@
 
 class Config(BaseConfig):
     DEBUG = False
-    LOG_LEVEL = 'DEBUG'
     LOG_CONFIG = 'kqueen/utils/logger_config.yml'
```

Comment:
Can you delete default `LOG_CONFIG` here or don't set in in `server.py` code. 

Please keep default on single place.

Reply:
project config doesn't set default values, it just keep all set up values in one place. But if you want, I'll delete LOG_CONFIG from all the configs

Reply:
I decided to remove hard coded value from, so it is parameter is required to be provided in a config file, otherwise error message about logging in set up correctly will be appear.